client/src/components/KeyConceptList/KeyConceptList.stories.tsx

```diff
@@ -0,0 +1,63 @@
+import { Story, Meta } from "@storybook/react";
+import React from "react";
+
+import { KeyConceptList, KeyConceptListProps } from "./KeyConceptList";
+
+export default {
+  title: "KeyConceptList",
+  component: KeyConceptList,
+} as Meta;
+
+const Template: Story<KeyConceptListProps> = (args) => (
+  <KeyConceptList {...args} />
+);
+
+const lorem_ipsum1 = (
+  <div>
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce quis risus
+    enim. Nulla a libero placerat, commodo purus at, rutrum nibh. Sed urna nunc,
+    eleifend ac hendrerit in, dictum sit amet nulla. Suspendisse quis egestas
+    ligula. Nunc mollis nibh ut ultricies ornare. Donec maximus feugiat eros, et
+    rhoncus risus. Pellentesque dignissim leo velit, vitae hendrerit diam
+    ultrices quis. Vivamus vulputate odio et orci vestibulum egestas.
+    Pellentesque accumsan neque in libero pharetra, et congue urna fermentum.
+    Aenean mattis sem ac erat interdum blandit.
+  </div>
+);
```

Comment:
Noticed this in some other files too, looks like it might be a prop type mistake in the actual component. This _should_ accept strings. Looks like the prop type uses `React.ReactElement` where it should be `React.ReactNode`. Double check that in the other components where you had to make this sort of change. If they work with strings, then please fix the prop types (whether or not the story uses strings or JSX may be unimportant, although it's easier to edit strings in controls than JSX).